sources/packages/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.controller.ts

```diff
@@ -51,21 +54,29 @@ export class ConfirmationOfEnrollmentController {
    * @returns student application list of an institution location
    */
   @HasLocationAccess("locationId")
-  @Get(":locationId/confirmation-of-enrollment")
+  @Get(
+    ":locationId/confirmation-of-enrollment/enrollmentPeriod/:enrollmentPeriod",
+  )
   async getCOESummary(
     @Param("locationId") locationId: number,
+    @Param("enrollmentPeriod") enrollmentPeriod: string,
   ): Promise<COESummaryDTO[]> {
-    const applications = await this.applicationService.getCOEApplications(
-      locationId,
-    );
-    return applications.map((eachApplication: Application) => {
+    const disbursementSchedules =
+      await this.disbursementScheduleService.getCOEByLocation(
```

Comment:
mm, i was just wondering, since `enrollmentPeriod` is getting  from URL, there are chances that user can edit it from URL, so, from `getCOEByLocation`, I see that, if the value is not `Upcoming` its fetching DB results. is that valid?

Reply:
IMO, if the value on enrollmentPeriod is not defined in the enum it must throw an 404 not found because the URL does no exists.

Reply:
API Updated to have only enum values in the the URI value.